app/src/main/res/layout/dialog_results.xml

```diff
@@ -249,10 +249,11 @@
                 android:textColor="@color/colorPrimary"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:text="▼ "
+                android:text="▼"
                 android:layout_gravity="center"
                 android:textSize="18sp"
-                android:clickable="true"/>
+                android:clickable="true"
+                android:paddingLeft="4dp" />
```

Comment:
Ah, the padding belong before and not after! Nice catch. And I agree that using layout instead of spaces is probably cleaner.